PlayCover/Utils/Cacher.swift

```diff
@@ -42,14 +68,14 @@ class Cacher {
         }
         cache.write(string: compareStr, forKey: compareStr)
         if let image = bestResImage {
-            cache.write(image: image, forKey: app.info.bundleIdentifier)
+            saveCachedImage(for: app.info.url, image: image)
         }
-        return cache.readImage(forKey: app.info.bundleIdentifier)
+        return getCachedImage(for: app.info.url)
     }
 
     func getLocalIcon(bundleId: String) -> NSImage? {
         if let app = AppsVM.shared.apps.first(where: { $0.info.bundleIdentifier == bundleId }) {
-            return cache.readImage(forKey: app.info.bundleIdentifier)
+            return getCachedImage(for: app.info.url)
```

Comment:
reasoning for changing identifier to link?

Reply:
Because on the documentation of the new `CachedAsyncImage`, to get a cached image i have to use 
```swift
private func getCachedImage(for url: URL) -> UIImage? {
        imageCache[url]
}
``` 
and `imageCache` is an array of URL and i cant change it to a String one

Reply:
Retrieving by URL is the proper method.

Reply:
this function is asking the installed apps for bundleids, then shows the cached image for the bundleid, if an installed app is present it will show the app icon from the installed one and not letting playcover to load from network in ipa library, plus if an app has no online icon and installed in playcover it will show the local installed app's app icon in ipa library, cachedasyncimage does anything about it i dunno but you are changing an intended behavior in playcover, please reconsider your changes.